app/src/main/java/com/m/androidNativeApp/CreateTask.java

```diff
@@ -0,0 +1,48 @@
+package com.m.androidNativeApp;
+
+import android.app.Activity;
+import android.content.Intent;
+import android.os.Bundle;
+import android.view.View;
+import android.widget.EditText;
+
+import com.apollographql.apollo.ApolloCall;
+import com.apollographql.apollo.api.Response;
+import com.apollographql.apollo.exception.ApolloException;
+
+import org.jetbrains.annotations.NotNull;
+
+public class CreateTask extends Activity {
+
+    protected void onCreate(Bundle savedInstanceState){
+        super.onCreate(savedInstanceState);
+
+        setContentView(R.layout.add_task);
+    }
+
+    public void backToMain(View view){
```

Comment:
the body of the function isn't really doing what the name has suggested here. Perhaps consider rename the method to make it more clear.

Reply:
Very good point, thank you.